erpnext_com/www/index.html

```diff
@@ -11,13 +11,17 @@
 	<meta name="author" content="erpnext.com">
 	<meta name="robots" content="index, follow">
 	<meta name="revisit-after" content="1 days">
-	<meta property="og:title" content="ERPNext - Open Source Cloud ERP Software">
-	<meta property="og:description" content="ERPNext is the world's top 100% open source ERP software. ERPNext supports manufacturing, distribution, retail, trading, services, education, non profits and healthcare">
-	<!-- <meta property="og:image" content="{{ frappe.utils.get_url() }}/assets/erpnext_com/img/home/social.png"> -->
-	<meta property="og:url" content="https://erpnext.org/conf/2019">
+	<!-- Open Graph Protocol -->
+	<meta property="og:title" content="ERPNext - Open Source Cloud ERP">
+	<meta property="og:description" content="Being 100% Open Source ERP, ERPNext supports manufacturing, distribution, retail, trading, services, education, non profits and healthcare">
```

Comment:
The sentence does not make sense. Starting with "Being 100%..." means the subsequent line is associated to it.

The previous line was better, besides the description does not have much SEO impact, make it compelling for users as it'll show up on the search listing

Reply:
I merely removed commented out words! Will update